react/daily-art-prompt/src/store/rootReducer.test.js

```diff
@@ -527,3 +527,39 @@ describe('indexReducer', () => {
         })
     })
 })
+
+describe('promptsReducer', () => {
+    describe('when action.type is SET_INITIAL_PROMPTS', () => {
+        it('should return previous state with new prompt', () => {
```

Comment:
either add something to state, or add a new test for that case

Reply:
done